go/libkb/api.go

```diff
@@ -494,7 +494,7 @@ func (a *InternalAPIEngine) fixHeaders(arg APIArg, req *http.Request) {
 		}
 	}
 	if a.G().Env.GetTorMode().UseHeaders() {
-		req.Header.Set("User-Agent", UserAgent)
+		req.Header.Set("User-Agent", UserAgent())
```

Comment:
Hah.  Or we could go back to the second version, which didn't have this problem and is now equally complex.